dangogh/trafficcontrol
# Add x509v3 Self-Signed CA and TLS certificates and misc DNS fixes

#### What does this PR do?

Add x509v3 Self-Signed CA and TLS certificates
Fix others issue with DNS related to ciab.test.

#### Which TC components are affected by this PR?


#### What is the best way to verify this PR?

1) rebuild images 
2) restart with docker-compose up
3) Wait for entire stack to load all data.
4) Go into delivery service and add the associate the "edge" server to the "demo1" delivery service.
5) Perform a snapshot
6) Perform a queue updates

Should have the ability to create unlimited number of delivery services for any domain at *.mycdn.ciab.test.

#### Check all that apply
